File: tournaments/growingPlant/growingPlant.cs
int growingPlant(int upSpeed, int downSpeed, int desiredHeight) {
    var result = 1;
    
    if (upSpeed >= desiredHeight) {
        return result;
    }
    
    var height = upSpeed;
    var diff = upSpeed - downSpeed;
    
    while (desiredHeight > height) {
        height += diff;
        ++result;
    }
    
    return result;
}

File: tournaments/growingPlant/growingPlant.js
function growingPlant(upSpeed, downSpeed, desiredHeight) {
    let result = 0;
    let a = 0;

    while (desiredHeight > a) {
        a += upSpeed;

        if (desiredHeight <= a) return result + 1;

        a -= downSpeed;
        ++result;
    }

    return result
}
